client/components/environment/Main/Main.js

```diff
@@ -45,14 +45,12 @@ export default function Main({ location }) {
         <ReactNotification />
         <div className="main">
           <Switch>
-            <Route exact path="/" component={WelcomePage} />
-            <PageLayout>
-              <Route path="/login" component={LoginPage} />
-              <Route path="/register" component={RegisterPage} />
-              <Route path="/home" component={HomePage} />
-              <Route path="/todo" component={TodoPage} />
-              <Route path="/settings" component={SettingsPage} />
-            </PageLayout>
+            <Route path="/" component={WelcomePage} />
```

Comment:
We need to keep the `exact` on the `/` otherwise the other routes won't work

Reply:
the main dosnt look like this anymore i will 

> ![image](https://user-images.githubusercontent.com/23039408/156717869-9b9fc8f6-5cf0-40ed-884b-5ca117aad671.png)
> The Iphone 8 view disnt include the page

fixed